docker/beta.Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM ubuntu:xenial as builder
 
 # Grab dependencies
-RUN apt update
-RUN apt dist-upgrade --yes
-RUN apt install --yes curl jq squashfs-tools
+RUN apt-get update && apt-get dist-upgrade --yes && apt-get install --yes \
```

Comment:
Can you change these back to individual RUN commands?

Reply:
It is advised to avoid running `apt-get update` and `apt-get install`. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Why do you want this?

Reply:
Reverted back to individual commands.